hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/client/TestCompactionAdminClient.java

```diff
@@ -283,7 +282,7 @@ private List<Pair<HoodieLogFile, HoodieLogFile>> validateUnSchedulePlan(Compacti
 
     // Ensure same number of log-files before and after renaming per fileId
     Map<String, Long> fileIdToCountsAfterRenaming =
-        newFsView.getAllFileGroups(HoodieTestUtils.DEFAULT_PARTITION_PATHS[0]).flatMap(HoodieFileGroup::getAllFileSlices)
+        newFsView.getLatestMergedFileSlicesBeforeOrOn(HoodieTestUtils.DEFAULT_PARTITION_PATHS[0], ingestionInstant)
             .filter(fs -> fs.getBaseInstantTime().equals(ingestionInstant))
```

Comment:
The 0.x fs view does not really filter out the uncommitted files, we aim to fix that first right?

Reply:
yes, i'll update the test. The main fix here is in `HoodieFileGroup`